hack/validate_cluster.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+set -xE
+
+trap display_status EXIT
+
+#############
+# FUNCTIONS #
+#############
+EXEC_COMMAND="kubectl -n rook-ceph exec $(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[0].metadata.name}') -- ceph --connect-timeout 3"
+
+function wait_for_daemon () {
+  timeout=90
```

Comment:
This timeout may make more sense shorter or longer for some daemons, but we can refine that later.